hardhat.config.ts

```diff
@@ -1,5 +1,7 @@
 import "@nomiclabs/hardhat-etherscan";
 import "@nomiclabs/hardhat-waffle";
+import "@typechain/hardhat";
+import "@nomiclabs/hardhat-ethers";
```

Comment:
`typechain-types` folder was not being built. Needed this so contracts had types